### PR TITLE
add angular-stripe@4.2.8

### LINKED ISSUE #6557: [Request] Add angular-stripe
**Library name:** angular-stripe
**Git repository url:** https://github.com/bendrucker/angular-stripe
**npm package url(optional):** angular-stripe
**License(s):** MIT
**Official homepage:** https://github.com/bendrucker/angular-stripe
**Wanna say something? Leave message here:**

=====================
Notes from cdnjs maintainer:
You are welcome to add a library via sending pull request,
it'll be faster then just opening a request issue,
and please don't forget to read the guidelines for contributing, thanks!!

### PR DESCRIPTION
**Pull request by Piicksarn**
@maruilian11 , would you mind to have a look at the pr?
The PR is referenced to issue  #6557
The git repo : https://github.com/bendrucker/angular-stripe

- [x] Not found on cdnjs repo
- [x] No already exist issue and PR
- [x] Notable popularity : **Watch `10`, Star `145`, Fork `18`**
- [x] Project has public repository on famous online hosting platform (or been hosted on npm) 
- [x] Has valid tags for each versions (for git auto-update)
- [x] Auto-update setup
- [x] Auto-update target/source is valid.
- [x] Auto-update filemap is correct.

Thank you!

### CODE REVIEW

File: ajax/libs/angular-stripe/package.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/bendrucker/angular-stripe.git"
+  },
+  "npmName": "angular-stripe",
+  "npmFileMap": [
+    {
+      "basePath": "release",
+      "files": [
+        " angular-stripe*.js"
```
Comment (PeterDaveHello) [inline ajax/libs/angular-stripe/package.json]: What's the leading space for?

**Comment by maruilian11** [comment]
@Piicksarn There seems no `release` in npm package? how do you confirm it?

**Comment by Piicksarn** [comment]
@maruilian11, would you mind to have a look again?
Thank you!

**Comment by maruilian11** [comment]
@Piicksarn sorry for late, please rebase the commit, thank you!

**Comment by Piicksarn** [comment]
@maruilian11, would you mind to have a look at the pr?
thank you!

File: ajax/libs/angular-stripe/package.json [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/bendrucker/angular-stripe.git"
+  },
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/bendrucker/angular-stripe.git",
+    "basePath": "release",
+    "files": [
+      "angular-stripe*.js"
```
Comment (PeterDaveHello) [inline ajax/libs/angular-stripe/package.json]: You can use `"**/*"` under `release`.

**Comment by PeterDaveHello** [comment]
Why don't you use the prefered npm auto-update?
https://github.com/bendrucker/angular-stripe#setup

**Comment by Piicksarn** [comment]
@PeterDaveHello ,
there is no js file which we in need when using npm auto-update.
i found that the release/angular-stripe.js only be included on github.

**Comment by PeterDaveHello** [comment]
But your commit message says: add angular-stripe@4.2.8 with npm autoupdate.
https://github.com/Piicksarn/cdnjs/commit/96a71ce30b0da892dcac03f8eb20cb08e16688f7

**Comment by PeterDaveHello** [comment]
Close per #7059

**Pull request closed without merging**